src/OSPSuite.UI/Views/Importer/ColumnMappingView.cs

```diff
@@ -344,15 +346,24 @@ public string Format(DataFormatParameter model)
 
          if (model is MappingDataFormatParameter mapping)
          {
-            var str = $"Units: {mapping.MappedColumn.Unit.SelectedUnit}";
+            var str = "";
```

Comment:
Suggestion. This allocates the old and new string all the time
I would recommend creating a list, adding error to the list and then using toString() `extension` method to create one string from an Ienumerable. 
In general += on string can be easily avoided

Reply:
Yeap I like this suggestion a lot. I changed this. Also changed class signature to internal.